rasterio/rio/warp.py

```diff
@@ -118,7 +118,7 @@ def warp(ctx, files, output, driver, like, dst_crs, dimensions, src_bounds,
                       CHECK_WITH_INVERT_PROJ=check_invert_proj):
         with rasterio.open(files[0]) as src:
             l, b, r, t = src.bounds
-            out_kwargs = src.meta.copy()
+            out_kwargs = src.profile.copy()
```

Comment:
A subtle but important enhancement since it holds onto the compression and block info 👍